src/renderer/containers/Onboard/CreatePassword/CreatePassword.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from "react";
+import {Component, ReactElement} from "react";
+import {IInputFormProps} from "../../../components/Input/InputForm";
+import {ButtonPrimary} from "../../../components/Button/ButtonStandard";
+import {History} from "history";
+import {MultipleInputVertical} from "../../../components/MultipleInputVertical/MultipleInputVertical";
+import passwordComplexity from "../../../services/validation/password-complexity";
+import {partition} from "../../../services/utils/utils";
+
+interface IState {
+    input: string;
+    confirmInput: string;
+    valid?: boolean;
+    validationMessage: string;
+    confirmed?: boolean;
+}
+
+const SEPARATOR = "and";
+
+export default class CreatePassword extends Component<{ history: History }, {}> {
```

Comment:
- no export default, it messes up import typehinting.
- `extends Component<Pick<RouteComponentProps, "history">>`
- CreatePasswordContainer

Reply:
Regarding the last point, is it really a separate container, because it is part of OnboardContainer?

Reply:
@mpetrunic

Reply:
It handles inputs and navigation and everything so it can't be component